mkaplan9/vote_pa
# Adds basic tests

# Summary 

This PR does the following...
- Moves the Puppeteer functions to a separate `utils.js` file. 
- Adds some basic tests to ensure that the Puppeteer functions are working as intended. 
- Adds some comments to the Puppeteer functions.
- Adds a github workflow to run the tests whenever there's a push to a branch. 

# Testing Instructions

Pull the branch, run `npm i` to install any new deps, then run `npm test`. The Puppeteer functions are still kinda brittle, so one of the tests may fail by chance, but it should pass 4/5 times. 

Will continue working on how to make the Puppeteer functions more resilient in a later PR.
